fe1-web/model/network/method/message/data/MessageRegistry.ts

```diff
@@ -80,9 +115,45 @@ export class MessageRegistry {
     }
     const { handle } = messageEntry;
     if (handle === undefined) {
-      console.warn(`Message ${data.object} ${data.action} does not have a 'handle' property in MessageRegistry`);
+      console.warn(`Message ${data.object} ${data.action} does not have an 'handle' property in MessageRegistry`);
```

Comment:
<grammar-nazi>
It was correct before. 
The "h" in "handle" is not silent, hence "a" 😄 
</grammar-nazi>